src/components/flow/flow.tsx

```diff
@@ -119,44 +97,21 @@ export class Flow {
     return newPanelCount < oldPanelCount ? "retreating" : "advancing";
   };
 
-  handleMutationObserverChange = (): void => {
-    const { el, allowDescendantPanels } = this;
-
-    if (!allowDescendantPanels) {
-      return;
-    }
+  updateFlowProps = (): void => {
+    const { el, panels } = this;
 
     const newPanels: HTMLCalcitePanelElement[] = Array.from(
-      el.querySelectorAll("calcite-panel:not(calcite-panel calcite-panel)")
-    );
-
-    this.updateFlowProps(newPanels);
-  };
-
-  handleDefaultSlotChange = (event: Event): void => {
-    if (this.allowDescendantPanels) {
-      return;
-    }
-
-    const newPanels = (event.target as HTMLSlotElement)
-      .assignedElements({
-        flatten: true
-      })
-      .filter((el) => el?.matches("calcite-panel")) as HTMLCalcitePanelElement[];
+      el.querySelectorAll("calcite-panel")
+    ).filter((panel) => !panel.closest("calcite-panel")) as HTMLCalcitePanelElement[];
```

Comment:
@jcfranco storybook kept failing when using `el.querySelectorAll("calcite-panel:not(calcite-panel calcite-panel)")`.

I'm guessing this is because its stuck on a version that doesn't support it?

https://developer.mozilla.org/en-US/docs/Web/CSS/:not#browser_compatibility

I'd rather use the single selector if possible but storybook is holding us back here.

Reply:
cc @eriklharper

Reply:
> I'd rather use the single selector if possible but storybook is holding us back here.

Yeah, that'd be cleaner. For now, let's proceed with this tested version and we should create a refactor issue to make the switch once newer browsers are used for screenshot tests.

Reply:
Yeah, unfortunately we'll have to just deal with the outdated Chrome for now until we can make the switch to Chromatic.